137_single_number.py

```diff
@@ -0,0 +1,27 @@
+"""
+https://leetcode.com/problems/single-number-ii/description/
+Given an array of integers, every element appears three times except for one, which appears exactly once. Find that single one.
+
+Note:
+Your algorithm should have a linear runtime complexity. Could you implement it without using extra memory?
+"""
+
+class Solution(object):
+    def singleNumber(self, nums):
+        """
+        :type nums: List[int]
+        :rtype: int
+        """
+        # For no extra memory, we'll go with the pop method
```

Comment:
Even the pop method has some memory usage. The sum method is the most memory efficient.